packages/core/src/environments/hardhat/upgradeable/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "hardhat-sample",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "npx hardhat test"
```

`npx` not necessary here. When you say `npm run foo`, the script `foo` will run with $PATH pointing to the dependency executables.

```suggestion
    "test": "hardhat test"
```